cilium/cmd/encrypt_status_test.go

```diff
@@ -119,3 +119,30 @@ func (s *EncryptStatusSuite) TestGetXfrmStats(c *C) {
 	}
 	c.Assert(currentCount, Equals, errCount)
 }
+
+func (s *EncryptStatusSuite) TestExtractMaxSequenceNumber(c *C) {
+	ipOutput := `src 10.84.1.32 dst 10.84.0.30
+	proto esp spi 0x00000003 reqid 1 mode tunnel
+	replay-window 0
+	mark 0x3cb23e00/0xffffff00 output-mark 0xe00/0xf00
+	aead rfc4106(gcm(aes)) 0x64ad37a9d8a8f20fb2e74ef6000f9d580898719f 128
+	anti-replay context: seq 0x0, oseq 0xc3, bitmap 0x00000000
+	sel src 0.0.0.0/0 dst 0.0.0.0/0
+src 0.0.0.0 dst 10.84.1.32
+	proto esp spi 0x00000003 reqid 1 mode tunnel
+	replay-window 0
+	mark 0xd00/0xf00 output-mark 0xd00/0xf00
+	aead rfc4106(gcm(aes)) 0x64ad37a9d8a8f20fb2e74ef6000f9d580898719f 128
+	anti-replay context: seq 0x0, oseq 0x1410, bitmap 0x00000000
+	sel src 0.0.0.0/0 dst 0.0.0.0/0
+src 10.84.1.32 dst 10.84.2.145
+	proto esp spi 0x00000003 reqid 1 mode tunnel
+	replay-window 0
+	mark 0x7e63e00/0xffffff00 output-mark 0xe00/0xf00
+	aead rfc4106(gcm(aes)) 0x64ad37a9d8a8f20fb2e74ef6000f9d580898719f 128
+	anti-replay context: seq 0x0, oseq 0x13e0, bitmap 0x00000000
+	sel src 0.0.0.0/0 dst 0.0.0.0/0`
+
+	maxSeqNumber := extractMaxSequenceNumber(ipOutput)
```

Comment:
nit: Per the above we could test call boundaries without bombing vs just happy path

Reply:
Yep, that's a good point. I'm however unsure of what a valid input that would cause a fatal would be. Any idea?

Reply:
> Tx Paul! I am just neat picking here. So please feel free to drop if too much noise...

Well... you did reject the PR based on those three nitpicks ;)

> Me thinks "anti-replay context: seq 0x0, oseq 0x, bitmap 0x00000000" will cause us to bomb ;(

But that's not something ip x states can return.

Reply:
Well you did ask for a scenario that would cause this call to bomb.
At the very least, we should check the returned slice bounds?

Reply:
But what you call bomb is erroring out from the CLI with a proper error message. Or am I missing something?

Reply:
You are right! My feel on these kind of calls is we should just return an error and let the call site decide. Best if fatal only occur at the top level. Also easier to test boundaries.
But that's only my opinion...

Reply:
I found one (I *think* realistic) case that I thought could cause a fatal (interrupted output) so I added it. Turns out it's correctly handled, but probably worth having just in case.